drf_kit/signals.py

```diff
@@ -40,3 +41,15 @@ def __exit__(self, *args, **kwargs):
 post_soft_delete = Signal(providing_args=["instance"])
 pre_undelete = Signal(providing_args=["instance"])
 post_undelete = Signal(providing_args=["instance"])
+
+
+@receiver(post_delete)
+def reorder_group(sender, instance, **kwargs):
+    from drf_kit.models import OrderedModelMixin  # pylint: disable=import-outside-toplevel
+
+    if not issubclass(sender, OrderedModelMixin):
+        return
+
+    if previous := instance.previous():
+        # trigger the reordering
+        previous.save()
```

Comment:
You can [register signals for all instances of this model](https://github.com/flamingo-run/drf-toolkit/blob/main/drf_kit/models/ordered_models.py#L59) instead of registering to all, and them filtering.